externals/nw.cppan~.mxo/Contents/Info.plist

```diff
@@ -1,42 +1,36 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<!DOCTYPE plist PUBLIC "-//Apple Computer//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
 <plist version="1.0">
 <dict>
-	<key>BuildMachineOSBuild</key>
-	<string>14D136</string>
 	<key>CFBundleDevelopmentRegion</key>
 	<string>English</string>
 	<key>CFBundleExecutable</key>
 	<string>nw.cppan~</string>
+	<key>CFBundleGetInfoString</key>
+	<string></string>
+	<key>CFBundleIconFile</key>
+	<string></string>
 	<key>CFBundleIdentifier</key>
-	<string>com.cycling74.nw-cppan-</string>
+	<string></string>
 	<key>CFBundleInfoDictionaryVersion</key>
-	<string>6.1.4</string>
+	<string>6.0</string>
 	<key>CFBundleLongVersionString</key>
-	<string>nw.cppan~ 6.1.4, Copyright 2013 Cycling '74</string>
```

Comment:
I did not notice in the initial transition that this copyright notice was added. Glad the max-sdk-core undoes this!